woocommerce/woocommerce
# Allow unsaved products to edit the product link

### All Submissions:






### Changes proposed in this Pull Request:

Allows unsaved products to edit the product link.  This works by saving any unsaved products as auto-drafts.

<img width="509" alt="Screen Shot 2023-01-27 at 12 33 10 PM" src="https://user-images.githubusercontent.com/10561050/215191550-cf92e5dd-d7fc-4960-967f-ab844b3a8f4b.png">


Part of #36317 .






### How to test the changes in this Pull Request:



1. Enable the new product experience feature
2. Navigate to the Products->Add Product page
3. Type in some text in the "Name" field
4. Note that the product link field appears and you can edit and save the product link
5. Publish the product and note all the data and link is saved correctly



### Other information:




### FOR PR REVIEWER ONLY:
